editoast/src/views/timetable.rs

```diff
@@ -114,7 +118,7 @@ struct TimetableIdParam {
 
 /// Return a specific timetable with its associated schedules
 #[utoipa::path(
-    get, path = "",
+    get, path = "/train_schedules",
```

Comment:
Does this really work (cc @leovalais)? I was under the impression that with our macro's system, paths need to be defined in `crate::routes!()` (the macro defines both OpenAPI and `axum`'s router, so you might be missing the `axum` part?). At least, when testing, `/timetable/1` returns the entire payload, but `/timetable/1/train_schedules` returns a 404.

Reply:
No it doesn't you're completely right, I should stop reviewing PRs late... 🤦 

It should indeed be `""` and put the path suffix in the `routes!` macro. Not sure how easy it would be, but triggering an error when the path is non-empty may be a good idea.